documentation/src/main/asciidoc/topics/proc_indexing_rebuilding.adoc

```diff
@@ -0,0 +1,27 @@
+[id='indexing_rebuilding-{context}']
+= Rebuilding Indexes
+Rebuilding an index reconstructs it from the data stored in the cache.
+You rebuild indexes when you change things like the definitions of indexed types or analyzers.
+Likewise, you might also need to rebuild indexes if they are deleted for some reason.
+
+[IMPORTANT]
+====
+Rebuilding indexes can take a long time to complete because the process takes place for all data in the grid.
```

Comment:
Could you include the following warning, since I forgot to add:

WARN: Rebuilding indexes also affect queries, that can return less results than expected.